### ISSUE TITLE
minDate and MaxDate absolutely useless

### ISSUE THREAD

**Issue by rodic66** [RemiAWE/ng-flat-datepicker#23]
Yes, when you configure those parameters in datepickerConfig Calendar grayed out those dates that are not in the range, true. But if you click on those grayed out dates picker selects those values and displays them in the input box, not cool.
See that past dates are grayed out
![Uploading ScreenShot001.jpg…]()
And when I click on one of those grayed out dates you see it is still allow me to select those
![Uploading ScreenShot002.jpg…]()

**Comment by JaviCastellvi** [comment]
Hello rodic66, have you been able to resolve the minDate and maxDate so that dates that are grayed out are not selectable?

**Comment by rodic66** [comment]
Yes, I did figure this out. Let me get to my office and I will show what I did.
